time_tracker_api/activities/activities_namespace.py

```diff
@@ -41,41 +43,49 @@
     activity_response_fields
 )
 
-activities_dao = create_dao()
+activity_dao = create_dao()
 
 
 @ns.route('')
 class Activities(Resource):
     @ns.doc('list_activities')
-    @ns.marshal_list_with(activity, code=200)
+    @ns.marshal_list_with(activity, code=HTTPStatus.OK)
     def get(self):
-        return []
+        """List all activities"""
+        return activity_dao.get_all(), HTTPStatus.OK
 
     @ns.doc('create_activity')
+    @ns.response(HTTPStatus.BAD_REQUEST, 'Bad request')
```

Comment:
If we are going to put a message, lets better put one customized, like:
> Invalid format or structure of the attributes of the activity

Also, it would be nice to specify that a CONFLICT may occur if there is already an activity with the same name
```
    @ns.response(HTTPStatus.CONFLICT, 'This activity already exists')
```